### PR TITLE
Update cheatsheet.md

### PR DESCRIPTION
**Pull request by sangam14**

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
@sangam14
Pls sign the CLA in order for the PR to be reviewed.

**Comment by sangam14** [comment]
@Arhell https://identity.linuxfoundation.org/projects/cncf any you help me with this ! what I need to fill up under Contact >
>Subject * & Message *

**Comment by Arhell** [comment]
@sangam14
https://github.com/kubernetes/community/blob/master/CLA.md
+ your email is on github and there must match

**Comment by sangam14** [comment]
@Arhell what I need to fill under CNCF individual agreement ! in title field ? 
Screenshot 2020-12-05 at 6 47 34 PM

**Comment by sangam14** [comment]
@Arhell Done !

**Review by bl-ue — APPROVED** [review]

**Comment by bl-ue** [comment]
@sangam14 why did you close this?  It's perfect; just what we need!!!

**Comment by bl-ue** [comment]
Brad doesn't seem to be active these days.
/cc @sftim @oomichi

**Comment by bl-ue** [comment]
@sangam14 sometimes PR reviews can take a long while.  See kubernetes/kubernetes#96920 which was waiting 17 days! 😄

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -390,6 +390,7 @@ Verbosity | Description
 `--v=2` | Useful steady state information about the service and important log messages that may correlate to significant changes in the system. This is the recommended default log level for most systems.
 `--v=3` | Extended information about changes.
 `--v=4` | Debug level verbosity.
+`--v=5` | Trace level verbosity.
```
Comment (oomichi) [inline content/en/docs/reference/kubectl/cheatsheet.md:393]: How to know this level is Trace?

Comment (bl-ue) [reply content/en/docs/reference/kubectl/cheatsheet.md:393]: @oomichi @sangam14
See:
https://github.com/kubernetes/community/blob/4eeba4d57bed01502cb09598a74d21671d4ee876/contributors/devel/sig-instrumentation/logging.md#L28
(not sure why GitHub isn't displaying contents 🙄)

> klog.V(5) - Trace level verbosity

Comment (bl-ue) [reply content/en/docs/reference/kubectl/cheatsheet.md:393]: `kubectl` uses `klog`.

Comment (sangam14) [reply content/en/docs/reference/kubectl/cheatsheet.md:393]: klog have flags from --v=0 to 5 . and 4 is missing from  official documentation !

Comment (bl-ue) [reply content/en/docs/reference/kubectl/cheatsheet.md:393]: @sangam14 I'm not sure what you mean&mdash;level for is documented at https://github.com/kubernetes/community/blob/4eeba4d57bed01502cb09598a74d21671d4ee876/contributors/devel/sig-instrumentation/logging.md#L28.

Comment (oomichi) [reply content/en/docs/reference/kubectl/cheatsheet.md:393]: @bl-ue 
Thanks for your reply, that is a perfect answer for me :-)

/lgtm

**Review by brianpursley — APPROVED** [review]

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-25389--kubernetes-io-master-staging.netlify.app/docs/reference/kubectl/cheatsheet/#kubectl-output-verbosity-and-debugging

**Comment by kbhawkey** [comment]
Thanks @sangam14 .
/approve

**Pull request merged**